pkgs/development/tools/kustomize/default.nix

```diff
@@ -2,7 +2,7 @@
 
 buildGoModule rec {
   pname = "kustomize";
-  version = "3.10.0";
+  version = "4.0.0";
```

Comment:
Looks like we're already at 4.0.1: https://github.com/kubernetes-sigs/kustomize/releases/tag/kustomize%2Fv4.0.1